### DISCUSSION TITLE
Version Tags at cerbos-sdk-java

### DISCUSSION THREAD

**Discussion by bbodensieck** [cerbos/cerbos#1683] [Q&A] [answered]
Hi folks,

I wanted to try out the new admin api feature in `cerbos-sdk-java` (thank you btw!) but I struggled to find the correct version.

In you [documentation](https://github.com/cerbos/cerbos-sdk-java#installation) in the repo you have this example
```
implementation("dev.cerbos:cerbos-sdk-java")
```
This did not work for me.

The Package in the Repo https://github.com/cerbos/cerbos-sdk-java/packages/954297 is from 2022-06-08.

Then I had a look in https://mvnrepository.com/artifact/dev.cerbos/cerbos-sdk-java and found that the newest version tag is `0.6.1`

Then I saw in the repo ![](https://camo.githubusercontent.com/94870c4c4f3459e739c005a06807426b60229d1f4820acb0131fa555c550e153/68747470733a2f2f696d672e736869656c64732e696f2f6d6176656e2d63656e7472616c2f762f6465762e636572626f732f636572626f732d73646b2d6a6176613f7374796c653d666f722d7468652d62616467652676657273696f6e5072656669783d302e) 
...but `v0.7.0` as a tag also did not work.

Then in https://central.sonatype.com/artifact/dev.cerbos/cerbos-sdk-java/47d60e3/versions I found several tags with different notations. `47d60e3`, `v0.5.0` and finally the correct version, which is `0.7.0`. 

So the example in the repo readme would be better with:

```
dependencies {
    implementation("dev.cerbos:cerbos-sdk-java:0.7.0")
}

repositories {
    mavenCentral()
}
```
Have a nice day and keep up the great work!
Björn

**Comment by charithe** [comment, marked as answer]
Hi Björn. Thanks for reporting. When you said that `implementation("dev.cerbos:cerbos-sdk-java")` didn't work for you, did you mean that it didn't download the correct version for you? AFAIK, not specifying the version should make Gradle download the latest version automatically (https://docs.gradle.org/current/userguide/single_versions.html#sec:declaring_without_version). This may be confounded by the fact that we have accidentally published one version as a Git hash instead of a proper version. I'll try to get that removed from Central if they allow that. In the mean time, you should be able to use `implementation("dev.cerbos:cerbos-sdk-java:0.+")` to grab the latest releases.

We used to use GitHub for distributing the package but switched to Maven Central later on. I suspect that's probably causing some confusion for installations that have both Maven Central and GitHub configured as package repositories. I have deleted the old GitHub repository so hopefully that will fix that issue going forward.

The Maven badge adds the `V` prefix to the version for some reason. I couldn't find a way to turn that off, unfortunately. I think they are using that cosmetically and expect the users to know that the Java convention is to ignore the `V` prefix 🤷🏽

**Reply by bbodensieck** [reply]
> [...] did you mean that it didn't download the correct version for you?

Yes exactly, it did not download any version. Maybe there is no file behind `latest`?

`implementation("dev.cerbos:cerbos-sdk-java:0.+")` works perfectly, thank you.
